src/confirmationModal/index.tsx

```diff
@@ -135,7 +147,13 @@ class ConfirmationModal extends Component<ConfirmationModalProps> {
         <TransitionGroup component="div" className="transition-wrapper">
           {isOpen && (
             <CustomTransition animationName={AnimationType.SLIDE_UP}>
-              <div className={classNames}>
+              <div
+                className={classNames}
+                role="alertdialog"
+                aria-labelledby={ariaLabelledBy}
+                aria-describedby={ariaDescribedBy}
+                aria-modal="true"
```

Comment:
FYI, last time I checked the screenreader support for aria-modal was really limited.
I'm not sure we want to aria-hidden background elements either (we used to do that in some past projects and it really confiuses some screenreaders).

Reply:
I checked with VoiceOver only, it didn't seem to be confused. You're right, the docs are saying that we should make the other elements "inert" which `aria-modal` does, the old way to do it is to trigger a `aria-hidden` on those elements manually, at least we're future proof

> The aria-modal property was introduced in ARIA 1.1. As a new property, screen reader users may experience varying degrees of support for it.
> Applying the aria-modal property to the dialog element replaces the technique of using aria-hidden on the background for informing assistive technologies that content outside a dialog is inert.